kedro_kubeflow/generators/pod_per_node_pipeline_generator.py

```diff
@@ -41,13 +42,13 @@ def generate_pipeline(self, pipeline, image, image_pull_policy):
         )
         @maybe_add_params(self.context.params)
         def convert_kedro_pipeline_to_kfp() -> None:
+            from kedro.framework.project import pipelines
```

Comment:
Move the import to the imports section. This in addition breaks the docstring commentage.